packages/cli/src/lib/setup.ts

```diff
@@ -277,18 +278,13 @@ execute_prompt_phase() {
         safe_exit 1 "Prompt file /prompts/$phase_name.txt not found"
     fi
 
-    # Switch to agent user and execute the prompt
-    su agent << EOF
-# Change to workspace directory
-cd /workspace
+    sudo chown -R $(id -u):$(id -g) $HOME
+    sudo chown -R $(id -u):$(id -g) /workspace
+    sudo chown -R $(id -u):$(id -g) /output
```

Comment:
This is the weird part to me. Is this only required on `rootless` mode or on both installations? For regular installations, the `/workspace` and `/output` folders should be the target user.

Reply:
Thanks for the clarification 👍 

I got it, the mount happens always as root in docker rootless. Users mappings goes after it.